### PR TITLE
feat: Add column visibility to table v8 [MDS-1374]

### PR DESCRIPTION
**Pull request by MarioGranada**

### CODE REVIEW

File: packages/table-v8/src/components/Table.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useRef, useState } from "react";
+import React, { useEffect, useRef, useState, forwardRef } from "react";
```
Comment (dkireev) [inline packages/table-v8/src/components/Table.tsx:1]: Remove it if unused

**Review by dkireev — CHANGES_REQUESTED** [review]

File: docs/app/client/icons/icons/IconWrapper.tsx [hunk 2]
```diff
@@ -62,7 +62,9 @@ const IconWrapper = ({ children, name }: IconProps) => {
       >
         <Snackbar.Message className="flex gap-2">
           <Icons.GenericCheckAlternative className="text-moon-24 text-roshi" />
-          Icon {renderIcon(name)} copied for import
+          Icon {renderIcon(
```
Comment (dkireev) [inline docs/app/client/icons/icons/IconWrapper.tsx:65]: Please, roll back the changes. It's not relevant to the ticket

File: docs/app/client/icons/search/IconSearch.tsx [hunk 3]
```diff
@@ -113,7 +113,9 @@ const IconSearch = () => {
       >
         <Snackbar.Message className="flex gap-2">
           <Icons.GenericCheckAlternative className="text-moon-24 text-roshi" />
-          Icon {renderIcon(lastClickedIcon)} copied for import
+          Icon {renderIcon(
```
Comment (dkireev) [inline docs/app/client/icons/search/IconSearch.tsx:116]: Please, roll back the changes. It's not relevant to the ticket

**Review by dkireev — CHANGES_REQUESTED** [review]

File: docs/app/client/icons/icons/IconWrapper.tsx (hunk 2 above)
Comment (MarioGranada) [reply docs/app/client/icons/icons/IconWrapper.tsx:65]: Agreed. Done.

File: docs/app/client/icons/search/IconSearch.tsx (hunk 3 above)
Comment (MarioGranada) [reply docs/app/client/icons/search/IconSearch.tsx:116]: Agreed. Done.

**Review by dkireev — APPROVED** [review]

**Pull request merged**